src/js/components/account/visualizations/sankey/components/SankeyFlow.jsx

```diff
@@ -26,7 +26,7 @@ export default class SankeyFlow extends React.Component {
         };
     }
 
-    componentWillReceiveProps(nextProps) {
+    componentDidUpdate(nextProps) {
         this.calculatePath(nextProps);
```

Comment:
componentDidUpdate has a different param than componentWillReceiveProps. It is actually the opposite param; prevProps.

In this case we want to  call `this.calculatePath()` with this.props.

```suggestion
        this.calculatePath(this.props);
```